src/google/protobuf/compiler/php/php_generator.cc

```diff
@@ -82,6 +82,10 @@ void GenerateEnumDocComment(io::Printer* printer, const EnumDescriptor* enum_,
                             int is_descriptor);
 void GenerateEnumValueDocComment(io::Printer* printer,
                                  const EnumValueDescriptor* value);
+void GenerateServiceDocComment(io::Printer* printer,
+                               const ServiceDescriptor* service);
+void GenerateMethodDocComment(io::Printer* printer,
```

Comment:
This should be `GenerateServiceMethodDocComment`, as just `method` is too ambiguous, as `GenerateFieldDocComment` also documents a method.